src/codegen.cpp

```diff
@@ -217,6 +217,11 @@ extern void _chkstk(void);
 //    return alloca(40960);
 //}
 #endif
+
+extern float __gnu_h2f_ieee(uint16_t param);
+extern float __extendhfsf2(uint16_t param);
+extern uint16_t __gnu_f2h_ieee(float param);
```

Comment:
```suggestion
extern double __gnu_f2h_ieee(uint16_t param);
```
(not that it matters here, since we just need the declared prototype)